src/sensor/sensor.cpp

```diff
@@ -25,16 +29,30 @@ Sensor::Sensor() :
         this->_connected = false;
         emit this->connectionUpdate();
     });
+
+    auto config = SettingsManager::self()->lastLinkConfiguration();
+    qCDebug(PING_PROTOCOL_SENSOR) << "Loading last configuration connection from settings:" << config;
+    _detector->appendConfiguration(config);
```

Comment:
The configuration we load here should be the first one that the detector checks (instant connection at startup). I suggest a public bool Detector::checkLink(LinkConfig& link) that will check a single link. Sensor can call this before calling Detctor::scan.

Reply:
Done.